app/classifier/feedback/feedback-summary-container.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+import counterpart from 'counterpart';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import FeedbackSummary from './feedback-summary';
+import * as feedbackActions from '../../redux/ducks/feedback';
+import processSingleFeedback from './process-feedback-single';
+import processDrawingFeedback from './process-feedback-drawing';
+
+const processFeedback = {
+  single: processSingleFeedback,
+  drawing: processDrawingFeedback
+};
+
+class FeedbackSummaryContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.generateFeedbackItems = this.generateFeedbackItems.bind(this);
+  }
+
+  componentWillMount() {
+    this.generateFeedbackItems();
```

Comment:
I think you can just call this in the class constructor and the React docs state doing this is preferred: https://facebook.github.io/react/docs/react-component.html#componentwillmount

FWIW, I'm not sure when `componentWillMount` would be preferred over the constructor, though this discussion suggests a use case is if a Redux dispatch needs to happen before mount: http://stackoverflow.com/questions/40828004/constructor-vs-componentwillmount-what-a-componentwillmount-can-do-that-a-const